src/core/light-controller.ts

```diff
@@ -12,11 +12,12 @@ import { localize } from '../localize/localize';
 
 export class LightController extends NotifyBase<LightController> implements ILightContainer {
     private _hass: HomeAssistant;
+    private _lightGroupEntity: LightContainer | undefined;
```

Comment:
Please rename this variable to `_lightGroup` (as it does not contain entity).
Also I'd prefer to use the syntax with question mark: `private _lightGroup?: LightContainer;`.